core/client/src/main/java/alluxio/client/file/options/LoadMetadataOptions.java

```diff
@@ -24,6 +25,7 @@
 @NotThreadSafe
 public final class LoadMetadataOptions {
   private boolean mRecursive;
```

Comment:
done. I didn't change the client side name but changed the server side naming since we will deprecate the client anyway